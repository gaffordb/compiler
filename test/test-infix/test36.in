let x : <int> = (ref 10) in
let y : <int> = (ref 0) in
  while (#x > 0) do
    x := (#x - 1) ; y := (#y + 1)
    end;
  #y
