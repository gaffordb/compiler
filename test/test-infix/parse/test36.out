(let x = (ref 10) in
(let y = (ref 0) in
((while (> (#x) 0) do
((x := (- (#x) 1)) ; (y := (+ (#y) 1)))
end) ; (#y))))
